sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/Models/NamespacedNameResource.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Class to refer resources which contains namespace and name. </summary>
+    public partial class NamespacedNameResource
+    {
+        /// <summary> Initializes a new instance of NamespacedNameResource. </summary>
+        public NamespacedNameResource()
```

Comment:
This should not end with resource since it is not a resource, CI is failing due to this.
Please consider to rename this